python/pyhie/allspark/model/qwen_v15.py

```
@@ -515,8 +583,9 @@ def _trans_weight(self, torch_weight, lora_name=None):
             quantize_mode = False if lora_name or key not in quantize_map else quantize_map[
                 key]  # lora量化的时候， 去掉lora_name判断即可
             if quantize_mode == False:
-                save_torch_to_allsparky(weights_path, key, tensor, mode,
-                                        split_mode, group_list)
+                _w(key, tensor, mode, split_mode, group_list)
+                if is_fp8 and scale_inv is not None:
+                    _w(key + ".scale_inv", scale_inv, mode, NOSPLIT, [])
```

Flake8 (7.3.0)

[error] 'NOSPLIT' may be undefined, or defined from star imports: ..quantization, .model_base, .quantization_utils

(F405)